vertical/Eta2Eta.F90

```diff
@@ -0,0 +1,1940 @@
+!#include "unused_dummy.H"
```

Comment:
What is going on here?  This is a very large new program, but looks to be copied from somewhere else (inferred by style).

Reply:
Yes, this this "SJ" code, in particular a version where Mike Manyin cleaned up the interface and did tests to make sure it conserved to the needs of the chemistry folks this would ultimately be fore. So yes, it is copied from somewhere else. 

The alternative would be to write our own conservative pressure regridding code. Doable and probably cleaner, but one more thing to do to get this all working.

Reply:
Thanks - more-or-less what I expected, but wanted to be certain.   Part of a large tier of low-level "FV" code that needs to be shared across multiple repos/orgs.